src/1AC760.c

```diff
@@ -195,7 +195,82 @@ ApiStatus GetActorLevel(Evt* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "1AC760", PartnerDamageEnemy);
 
-INCLUDE_ASM(s32, "1AC760", PartnerAfflictEnemy);
+ApiStatus PartnerAfflictEnemy(Evt* script, s32 isInitialCall) {
+    BattleStatus* battleStatus = &gBattleStatus;
+    Bytecode* args = script->ptrReadPos;
+    Actor* actor = get_actor(script->owner1.enemyID);
+
+    s32 returnValue;
+    s32 flags;
+    u8 statusChance;
+    s32 damageResult;
+
+    returnValue = *args++;
+    battleStatus->currentAttackElement = *args++;
+    battleStatus->currentAttackEventSuppression = *args++;
+    battleStatus->currentAttackStatus = *args++;
+    battleStatus->currentAttackStatus |= evt_get_variable(script, *args++);
+    battleStatus->currentAttackDamage = evt_get_variable(script, *args++);
+    battleStatus->powerBounceCounter = 0;
+
+    flags = *args++;
+    if ((flags & 0x30) == 0x30) {
+        battleStatus->flags1 |= 0x30;
+    } else {
```

Comment:
This should be able to be mde into a series of if else/if instead of having this nested part after

Reply:
Done; will push change in a bit.